### PR TITLE
Stepper style updates

### PR DESCRIPTION
**Pull request by kyle1morel**
# Description

Stepper style updates
Intake validation fixes
Intake wording updates
Bug fix (?) for auto save triggering on older submitted intakes

https://apps.nrs.gov.bc.ca/int/jira/browse/PADS-398

## Types of changes

Bug fix (non-breaking change which fixes an issue)
New feature (non-breaking change which adds functionality)

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by qhanson55 — COMMENTED** [review]
image
unused import

File: frontend/src/components/housing/submission/SubmissionIntakeForm.vue [hunk 1]
```diff
@@ -259,7 +259,7 @@ async function onLatLongInputClick() {
 
 async function onInvalidSubmit() {
   switch (validationErrors.value[0]) {
-    case IntakeFormCategory.CONTACTS:
+    //case IntakeFormCategory.CONTACTS:
```
Comment (qhanson55) [inline frontend/src/components/housing/submission/SubmissionIntakeForm.vue:262]: any reason to leave this?

File: frontend/tests/unit/components/shasIntake/EnquiryIntakeForm.spec.ts
Comment (qhanson55) [inline frontend/tests/unit/components/shasIntake/EnquiryIntakeForm.spec.ts:1]: Unused type import

File: frontend/tests/unit/components/shasIntake/SubmissionIntakeForm.spec.ts
Comment (qhanson55) [inline frontend/tests/unit/components/shasIntake/SubmissionIntakeForm.spec.ts:1]: unused contactPreference import

File: frontend/tests/unit/components/shasIntake/SubmissionIntakeForm.spec.ts [hunk 2]
```diff
@@ -144,18 +144,17 @@ describe('SubmissionIntakeForm', () => {
...
-          firstName: 'testFirstName',
-          lastName: 'testLastName',
-          phoneNumber: '2501234567',
-          email: 'test@test.com',
-          contactApplicantRelationship: ProjectRelationship.AGENT,
-          contactPreference: ContactPreference.PHONE_CALL
-        }
-      ]
-    });
+    // Contacts are kinda whack right now
```
Comment (qhanson55) [inline frontend/tests/unit/components/shasIntake/SubmissionIntakeForm.spec.ts:146]: reason to keep?

File: frontend/tests/unit/components/shasIntake/SubmissionIntakeForm.spec.ts [hunk 3]
```diff
@@ -200,36 +199,40 @@ describe('SubmissionIntakeForm', () => {
...
+    // const applicantTestFail = submissionIntakeSchema.validate({
+    //   contactFirstName: '',
```
Comment (qhanson55) [inline frontend/tests/unit/components/shasIntake/SubmissionIntakeForm.spec.ts:228]: needed?

**Review by sanjaytkbabu — APPROVED** [review]

File: frontend/src/components/housing/submission/SubmissionIntakeForm.vue (hunk 1 above)
Comment (kyle1morel) [reply frontend/src/components/housing/submission/SubmissionIntakeForm.vue]: Removed

File: frontend/tests/unit/components/shasIntake/EnquiryIntakeForm.spec.ts
Comment (kyle1morel) [reply frontend/tests/unit/components/shasIntake/EnquiryIntakeForm.spec.ts:1]: Removed

File: frontend/tests/unit/components/shasIntake/SubmissionIntakeForm.spec.ts
Comment (kyle1morel) [reply frontend/tests/unit/components/shasIntake/SubmissionIntakeForm.spec.ts:1]: Removed

File: frontend/tests/unit/components/shasIntake/SubmissionIntakeForm.spec.ts (hunk 2 above)
Comment (kyle1morel) [reply frontend/tests/unit/components/shasIntake/SubmissionIntakeForm.spec.ts:146]: It should be there, but the way contacts are currently being handled in the forms currently makes validating that one bit a pain. This needs to go back in at some point.

File: frontend/tests/unit/components/shasIntake/SubmissionIntakeForm.spec.ts (hunk 3 above)
Comment (kyle1morel) [reply frontend/tests/unit/components/shasIntake/SubmissionIntakeForm.spec.ts:228]: It should be there, but the way contacts are currently being handled in the forms currently makes validating that one bit a pain. This needs to go back in at some point.

**Review by qhanson55 — APPROVED** [review]

**Pull request merged**